xml/depl_cisco_opflex.xml

```diff
@@ -0,0 +1,475 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE appendix
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+<!-- Converted by suse-upgrade version 1.1 -->
+<appendix xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="app.deploy.cisco_opflex">
+ <title>Cisco ACI Integration with Opflex ML2 and GBP Drivers</title>
+ <info>
+<dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+    <dm:maintainer>fs</dm:maintainer>
+    <dm:status>editing</dm:status>
+    <dm:deadline/>
+    <dm:priority/>
+    <dm:translation>no</dm:translation>
+    <dm:languages/>
+</dm:docmanager>
+</info>
+<para/>
+<sect1 xml:id="sec.deployment">
+  <title>Deployment Into a Cisco ACI Fabric</title>
+
+  <para>
+ SUSE customers who wish to deploy &cloud; into a Cisco Application Centric Infrastructure (ACI) fabric environment can do this by using Cisco OpFlex ML2 or GBP drivers. Cisco ACI options are selected in the Neutron barclamp, which will make your &cloud; installation a member of your Cisco Application Policy Infrastructure Controller (APIC). The drivers are packaged as RPMs for easy installation. This is a technology preview. We have successfully tested these steps on systems directly connected to a switch port on the ACI fabric. After you have set this up you should be able to:
+  </para>
+
+  <itemizedlist mark="bullet" spacing="normal">
+   <listitem>
+    <para>
+    Create &ostack; system objects in the Cisco APIC.
+    </para>
+   </listitem>
+   
+   <listitem>
+    <para>
+     Create &cloud; tenants and networks in the Cisco APIC using the Horizon dashboard.
+    </para>
+   </listitem>
+   
+   <listitem>
+    <para>
+     Create a VXLAN-based network encapsulation in the Cisco ACI fabric.
+    </para>
+   </listitem>
+   
+   <listitem>
+    <para>
+     Start &slsa; instances on different compute nodes that can ping their gateway and each other.
+    </para>
+
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ 
+ <sect1 xml:id="sec.prereqisites">
+  <title>Prequisites</title>
+    <para>
+     As the Cisco ACI fabric is customized and highly-variable per customer's needs, you must have your ACI fabric already set up and working, and you need your own ACI expert administrator.
```

Comment:
We usually use 79 character-lines.
You can try `daps-xmlformat -i SOURCEFILE.xml` to get our "official" style in an automated fashion.
(of course, be careful, if you apply this to established chapters, you may introduce far more changes than useful, but since this is new, it should be fine.)